### DISCUSSION TITLE
Daily 22/05

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [NataliaAlvarezIspc/proyecto-ispc-ecommerce#394] [Daily Sprint]
Con respecto al proyecto:

- ¿Qué hice ayer?
- ¿Qué haré hoy?
- Impedimentos o dificultades

**Comment by rpgrca** [comment]
Hoy habría que empezar con el siguiente sprint, voy a empezar a agarrar cosas que hay reportadas y tal vez averiguar cómo se hacen los controladores para empezar a conectar django con angular porque por lo general habilitan la documentación un día después de mi clase.

**Comment by NataliaAlvarezIspc** [comment]
Ya estuve asignandome algunas de las que se armaron el fin de semana. También estuve arreglando lo de la lupa que se desfazaba en un determinado tamaño de pantalla. Además estoy tratando de encontrar una manera para que lo que devuelva no amplie la barra de navegación y ademas lo devuelva en orden. 
Voy a seguir trabajando con eso y, además,  voy a estar creando las tareas para el nuevo sprint

**Comment by BraianTroncoso** [comment]
El finde estuve soluciando unos bugs, hoy quizas me asgine un par mas. Tengo clases de Python, y tengo que ponerme al dia con unos tps de otra tec.
